ovip-0014.md

```diff
@@ -38,18 +38,25 @@ Initiator (Originator VASP)
 | Version     |                                      | `version`     | String       | 1..1  | fixed value `1.0`   |
 | Originator  | see [ovip-0013](https://github.com/OpenVASP/ovips/blob/master/ovip-0013.md#12-originator) (1.2) for elements | `originator`  |              | 1..1  |                     |
 | Beneficiary | see [ovip-0013](https://github.com/OpenVASP/ovips/blob/master/ovip-0013.md#13-beneficiary) (1.3) for elements | `beneficiary` |              | 1..1  |                     |
-| Transfer    |                                      | `transfer`    |              | 1..1  |                     |
-|             | Virtual asset type                   | `va`          | String       | 1..1  |                     |
-|             | Transfer type                        | `ttype`       | dt.TFR.ttype | 1..1  |                     |
-|             | Amount                               | `amount`      | Decimal      | 1..1  | 18 digits precision |
-| Comment     |                                      | `comment`     | String       | 0..1  |                     |
+| Transfer    |                                      | `transfer`    |                    | 1..1  |                     |
+|             | Virtual asset identifier             | `vaid`        | String             | 1..1  |                     |
+|             | Virtual asset identifier scheme      | `vaidscheme`  | dt.TFR.vaidscheme  | 1..1  |                     |
```

Comment:
On first sight I found "scheme" confusing. Maybe "class" or "type" is a better fit? Also, here I'd improve readability a bit
```
vaIdClass
virtualAssetType
```
. What do you think?

Reply:
What we want to say here is what identification scheme does vaId refer to. It is not uncommon to speak of a scheme in this context but you can also say vaIdType (or the long version camel case) of course

Reply:
Agree on CamelCase?

Reply:
Are we saying virtualAssetIdentifierType?